File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().split(' ').map(Number);

input.sort((a, b) => a - b);

var result = 0;

if(input[0] === input[1] && input[1] === input[2]) {
    result = 10000 + input[0] * 1000;
} else if(input[0] === input[1] || input[1] === input[2]) {
    result = 1000 + input[1] * 100;
} else {
    result = input[2] * 100;
}

console.log(result);

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
A, B, C = map(int, input().split())

# 만약 A, B, C가 모두 같다면 10000+같은눈x1,000

# 만약 A, B, C 중 2개만 같다면 1000+같은눈x100

# 다 다르다면 그중가장큰눈x100

if A == B and B == C and A == C:
  print(10000+A*1000)
elif (A == B and A != C) or (A == C and B != C):
  print(1000+A*100)
elif (B == C and A != B):
  print(1000+B*100)
else:
  list = [A, B, C]
  print(max(list)*100)